cli/command/context/list.go

```diff
@@ -61,39 +60,19 @@ func runList(dockerCli command.Cli, opts *listOptions) error {
 		if kubernetesEndpoint != nil {
 			kubEndpointText = fmt.Sprintf("%s (%s)", kubernetesEndpoint.Host, kubernetesEndpoint.DefaultNamespace)
 		}
+		if rawMeta.Name == command.DefaultContextName {
+			meta.Description = "Current DOCKER_HOST based configuration"
+		}
 		desc := formatter.ClientContext{
 			Name:               rawMeta.Name,
-			Current:            rawMeta.Name == curContext,
+			Current:            rawMeta.Name == curContext || (curContext == "" && rawMeta.Name == command.DefaultContextName),
```

Comment:
Thinking if it would be clearer to move this to an `if`, and use an intermediate variable (e.g. `isCurrent`) for this

Reply:
fixed